ocaml/idl/datamodel.ml

```diff
@@ -663,6 +663,8 @@ let _ =
     ~doc:"The specified VBD device is not recognized: please use a non-negative integer" ();
   error Api_errors.vm_not_resident_here [ "vm"; "host" ]
     ~doc:"The specified VM is not currently resident on the specified host." ();
+  error Api_errors.vm_not_resident [ "vm" ]
+    ~doc:"The specified VM is not currently resident on any host." ();
```

Comment:
This is odd, because neither of them sounds wrong to me (most such questions have a clear answer even if I can't articulate the reason). However, I think I would say "not currently". Note also that the line above already uses "not currently".